spec/provider-spec.js

```diff
@@ -69,4 +69,28 @@ describe("provider model", function() {
       expect(model.replacedCucumberRegex(step)).toEqual("${1:textArgument}\" can add text \"${1:textArgument}\" for \"${1:textArgument}\" hours and \"${1:textArgument}\" minutes");
     });
   });
+
+  describe('file reading', () => {
+    it('should read files directly from the directory provided', () => {
+      const rootPath = process.cwd();
+      const featureFilePaths = model.searchForPattern(`${rootPath}/spec/features`, /test\.feature/);
+      expect(featureFilePaths.length).toEqual(1);
+      const matchesExpected = featureFilePaths[0].match(/test\.feature/);
+      expect(!!matchesExpected).toEqual(true);
```

Comment:
Perhaps use `Boolean(matchesExpected)` here (and line 87) instead to reduce obscurity. WDYT?